Documentation/faq.md

```diff
@@ -1,5 +1,31 @@
 ## Frequently Asked Questions (FAQ)
 
+### etcd, Distributed Systems
+
+#### Follower, leader in etcd
+
+[Raft][raft] is leader-based, and the leader handles all client requests. Client does not need to know which node is the leader. Request to follower is automatically forwarded to leader. Raft algorithm divides time into terms, and each term begins with an election to choose one leader. If leader becomes unavailable, cluster immediately elects a new leader or automatically transfers the leadership to a follower. This requires no human intervention.
```

Comment:
I think the client can send request to all members part is more important. leader/follow part falls into the domain of raft, which should not be covered in etcd doc a lot.